src/sentry/api/endpoints/organization_events.py

```diff
@@ -16,11 +16,17 @@
     raw_query,
     SnubaTSResult,
 )
+from sentry import features
+
+from sentry.api.endpoints.organization_events_v2 import OrganizationEventsV2Endpoint
 
 
 class OrganizationEventsEndpoint(OrganizationEventsEndpointBase):
 
     def get(self, request, organization):
+        if features.has('organizations:events-v2', organization, actor=request.user):
+            return OrganizationEventsV2Endpoint().get(request, organization)
```

Comment:
I don't really like instantiating the endpoint here. Could we just redirect instead?
`redirect('<new_view_name>', foo='bar')`